### PR TITLE
update: 30.0.0

### PR DESCRIPTION
**Pull request by terop**
## Description of the change

Take latest version in use. Chart major version changed because of the updated major version of Nextcloud.

## Benefits

Users get latest features and fixes.

## Possible drawbacks

There may be bugs and / or regressions.

## Checklist 

- [X] I have read the [CONTRIBUTING.md](https://github.com/nextcloud/helm/blob/main/CONTRIBUTING.md#pull-requests) doc.
- [X] DCO has been [signed off on the commit](https://docs.github.com/en/github/authenticating-to-github/signing-commits).
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).

### CODE REVIEW

File: charts/nextcloud/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: nextcloud
-version: 5.5.6
-appVersion: 29.0.6
+version: 5.6.0
+appVersion: 30.0.0
```
Comment (provokateurin) [inline charts/nextcloud/Chart.yaml:4]:
```suggestion
version: 6.0.0
appVersion: 30.0.0
```
Last time a lot of people complained we didn't make a breaking change, so let's make a breaking change for this major update.

**Review by provokateurin — CHANGES_REQUESTED** [review]

File: charts/nextcloud/Chart.yaml (hunk 1 above)
Comment (terop) [reply charts/nextcloud/Chart.yaml]: Changed as suggested.

**Review by provokateurin — APPROVED** [review]
Awesome, thank you!

**Comment by twistedgrim** [comment]
Thank you guys for making it a major version change. This helped me know to look closer for the major version change for nextcloud <3 It really helps for ensuring DB migrations etc work properly and we take a closer look as we use gitops automation.

**Pull request merged**